### DISCUSSION TITLE
PHPMailer with POST variables = NO

### DISCUSSION THREAD

**Discussion by Villmer** [PHPMailer/PHPMailer#3046] [Q&A]
Hard coded variables obviously work fine:'
    $mail->setFrom("mail@company_a");
    $mail->addAddress("mail@company_b"); 
---
This, however, will NOT work:

$from_email = $_POST['from_email'];
$to_email = $_POST['to_email'];
$mail->setFrom($from_email); //nor $_POST['from_email']
$mail->addAddress($to_email); //nor $_POST['to_email']

**Comment by Synchro** [comment]
Nope. That code will work fine if `$_POST` contains those values. Check the contents of your variables.
